test/georef_ocd_mapping_t.cpp

```diff
@@ -0,0 +1,261 @@
+/*
+*    Copyright 2018 Libor Pecháček
+*
+*    This file is part of OpenOrienteering.
+*
+*    OpenOrienteering is free software: you can redistribute it and/or modify
+*    it under the terms of the GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    OpenOrienteering is distributed in the hope that it will be useful,
+*    but WITHOUT ANY WARRANTY; without even the implied warranty of
+*    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*    GNU General Public License for more details.
+*
+*    You should have received a copy of the GNU General Public License
+*    along with OpenOrienteering.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <algorithm>
+#include <vector>
+
+#include <QByteArray>
+#include <QLatin1Char>
+#include <QList>
+#include <QMetaType>
+#include <QObject>
+#include <QStaticStringData>
+#include <QString>
+#include <QStringList>
+#include <QStringLiteral>
```

Comment:
Private implementation detail of QString. Do not include. (May need an update of `iwyu-qt.imp`.)

(It doesn't even exist on my computer.)